packages/core/src/app/analytics/AnalyticsContext.ts

```diff
@@ -0,0 +1,26 @@
+import { CheckoutPaymentMethodExecutedOptions } from '@bigcommerce/checkout-sdk';
+import { createContext } from 'react';
+
+export interface AnalyticsEvents {
```

Comment:
You should be able to get this interface from SDK?

Reply:
`AnalyticsEvents` it is interface for available methods in `AnalyticsContext`
This events are not the same as we have in SDK, because one `AnalyticsEvents` can trigger one or two events from different analytics services from sdk and most of this events are not the same for different services.